pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "poetry-partifact-plugin"
+version = "0.1.0"
+description = "A poetry plugin that configures AWS CodeArtifact"
+authors = ["AminoEngineers <eng@amino.com>"]
+readme = "README.md"
+packages = [
+    { include = "poetry_partifact_plugin" },
+]
+homepage = "https://github.com/aminohealth/poetry-partifact-plugin"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+partifact = "^0.2.0"
+
+[tool.poetry.group.dev.dependencies]
+twine = "^4.0"
+setuptools = "^67.6"
+wheel = "^0.40.0"
+black = "^23.3.0"
+pytest = "^7.3.1"
+poetry = "^1.4"
+pytest-mock = "^3.10.0"
+
+[build-system]
+requires = ["poetry-core>=1.3.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
+profile = "black"
+
+[tool.poetry.plugins."poetry.application.plugin"]
+poetry-partifact-plugin = "poetry_partifact_plugin.plugin:PartifactPlugin"
+
+[virtualenvs]
+create = true
+in-project = true
```

Comment:
neat! In the past I've relied on a `poetry.toml` file to configure these options.